frontend/model/getters.js

```diff
@@ -255,28 +269,36 @@ const getters: { [x: string]: (state: Object, getters: { [x: string]: any }) =>
       return payments.sort((paymentA, paymentB) => paymentB.height - paymentA.height)
     }
   },
-  ourPayments (state, getters) {
-    const periodPayments = getters.groupPeriodPayments
-    if (Object.keys(periodPayments).length === 0) return
-    const ourIdentityContractId = getters.ourIdentityContractId
-    const cPeriod = getters.currentPaymentPeriod
-    const pPeriod = getters.periodBeforePeriod(cPeriod)
-    const currentSent = getters.ourPaymentsSentInPeriod(cPeriod)
-    const previousSent = getters.ourPaymentsSentInPeriod(pPeriod)
-    const currentReceived = getters.ourPaymentsReceivedInPeriod(cPeriod)
-    const previousReceived = getters.ourPaymentsReceivedInPeriod(pPeriod)
+  ourPaymentsReceivedInPeriod (state, getters) {
+    return (period) => getters.ourPaymentsReceivedInPeriodForGroup(getters.currentGroupState, period)
+  },
+  ourPaymentsForGroup (state, getters) {
+    return (state) => {
+      const periodPayments = getters.groupPeriodPayments
```

Comment:
Please check all of these `ForGroup` getters for uses of getters that do not use the `state` but instead still use the `currentGroupState` like this one.

Reply:
Good catch